### PR TITLE
paramiko

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -406,8 +406,12 @@ build_dpkg() {
 	fi
 
 	# Make a programatically generated version for this build
-	DATESTAMP=$(date +%Y%m%d%H%M%S)
-	chroot ${DPKG_OVERLAY} /bin/bash -c "cd $srcdir && dch -b -M -v ${DATESTAMP}~truenas+1 --force-distribution --distribution bullseye-truenas-unstable 'Tagged from truenas-build'" || exit_err "Failed dch changelog"
+	if [ "$name" -ne "paramiko" ] ; then
```
Comment (william-gr) [inline scripts/build.sh:409]: Hard-coding a package name here? Why? Thats sounds wrong :P

Comment (themylogin) [reply scripts/build.sh:409]: Unfortunately I am not very familiar with the strange programming language this code is written in 😞

Comment (william-gr) [reply scripts/build.sh:409]: Why do we need not to change paramiko version?

Alternatively we could have a flag in the manifest to not do that?

Thoughts @kmoore134 ?

Comment (themylogin) [reply scripts/build.sh:409]: Otherwise `dch` complains and renames source directory, and I am not able to chdir there via `subdir` parameter anymore

Comment (kmoore134) [reply scripts/build.sh]: The update you did to make a generic "generate_version" looks like how I would have solved it. Abstracted away so there is no hard-coded name, and we can re-use it if other packages require it later.

**Pull request merged**